### PR TITLE
Adding /wipebandata

### PR DESCRIPTION
**Pull request by ghost**
PR For Issue: https://github.com/TotalFreedom/TotalFreedomMod/issues/806

NOTE: I didn't found a duplicate of this PR.
https://github.com/Adamx120/TotalFreedomMod-Contributing/commit/93fcb98ea9beddc253a159f77613fbf85454d1e6 - Final Commit

### CODE REVIEW

**Comment by JeromSar** [comment]
@Adamx120 This PR has several problems, but most importantly it has not been tested, yet alone compiles. Additionally, this PR does not resolve the associated issue.

**Comment by Adamx120** [comment]
@JeromSar Fixed, Tested, Reviewed, Worked

**Comment by JeromSar** [comment]
@Adamx120 No this PR still does not compile, nor resolve the associated issue.

**Comment by Adamx120** [comment]
@JeromSar It compiles for me

**Comment by JeromSar** [comment]
Reopening due to duplicate PR (same functionality & same author).

**Comment by Adamx120** [comment]
@JeromSar Now, the command is fine?

**Comment by JeromSar** [comment]
@Adamx120 No, this PR still has all of the above mentioned problems.

**Comment by Adamx120** [comment]
@JeromSar Command fixed
Look under this comment

**Comment by JeromSar** [comment]
@Adamx120 Maybe you don't understand. To clarify, this issue had (and still has) the following problems:
* The PR does not compile.
* The PR does not resolve the associated issue.
* This PR disregards formatting standards.

**Comment by Adamx120** [comment]
in my netbeans it compiles

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_wipebandata [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+@CommandPermissions(level = AdminLevel.SENIOR, source = SourceType.ONLY_CONSOLE)
+@CommandParameters(description = "Cleaning all banlist excluding Perm-Ban list", usage = "/<command>")
+public class Command_wipebandata extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        TFM_Util.adminAction(sender.getName(), "Wiping all bans from the server", true);
+        {
+        TFM_BanManager.purgeIpBans();
```
Comment (OnlineRepz) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_wipebandata]: Formatting

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_wipebandata [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+@CommandPermissions(level = AdminLevel.SENIOR, source = SourceType.ONLY_CONSOLE)
+@CommandParameters(description = "Cleaning all banlist excluding Perm-Ban list", usage = "/<command>")
+public class Command_wipebandata extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        TFM_Util.adminAction(sender.getName(), "Wiping all bans from the server", true);
+        {
```
Comment (OnlineRepz) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_wipebandata]: Not needed

**Comment by OnlineRepz** [comment]
And also, you missing the .java extension.

**Comment by Wild1145** [comment]
The PR still does not resolve the original issue. The formatting also needs to be looked at as it looks off to me.

**Comment by PacksGamingHD** [comment]
we have /tfbanlist purge and /tfipbanlist purge

**Comment by RobinGall2910** [comment]
^^ why is this needed

**Comment by Wild1145** [comment]
@JeromSar Re-Open

**Review by Wild1145 — COMMENTED** [review]
This PR does not resolve the issue that it is associated with. Please check the original issue and amend your PR accordingly.

**Pull request closed without merging**